api/institutions/views.py

```diff
@@ -581,6 +582,53 @@ def get_default_search(self):
         )
 
 
+class _NewInstitutionSummaryMetricsList(JSONAPIBaseView, generics.RetrieveAPIView, InstitutionMixin):
+    '''list view for institution-summary metrics
```

Comment:
either keep it an actual list view (which should be easy with `ElasticsearchListView`, as your copy-paste included `get_default_search`) or update to stop calling it a list view (both in class-name and docstring)

this sort of inconsistency is actively (tho subtly) hostile to future contributors